mobile_files/package.json

```diff
@@ -61,8 +62,7 @@
     "react-native-udp": "https://github.com/status-im/react-native-udp.git#2.3.1-1",
     "react-native-webview-bridge": "git+https://github.com/status-im/react-native-webview-bridge.git#0.33.16-1",
     "react-navigation": "^2.12.1",
-    "react-native-safe-area-view": "0.9.0",
-    "realm": "2.15.3",
+    "realm": "^2.16.1",
```

Comment:
No ^ we need fixed releases for reproducible builds.
Also they release a minor version in the past without android support

Reply:
fixed